### ISSUE TITLE
Support the new ollama embedding model

### ISSUE THREAD

**Issue by jsrdcht** [logancyang/obsidian-copilot#466]
**Describe the solution you'd like**

Just let users choose the embedding models from lists of ollama. Now only the ollama-nomic-embed-text is available.

**Comment by huyz** [comment]
Shouldn't this be closed as completed?

**Comment by konstantin1722** [comment]
> Shouldn't this be closed as completed?

Yeah, it kinda already supports selecting local embedding models from Ollama.
